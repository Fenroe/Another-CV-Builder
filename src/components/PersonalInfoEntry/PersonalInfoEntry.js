function PersonalInfoEntry() {
    return (
        <div>
            <fieldset>
                <legend>Personal Details</legend>
                <label>First Name</label>
                <input />
                <label>Last Name</label>
                <input />
                <label>Email Adress</label>
                <input />
                <label>Phone Number</label>
                <input />
                <label>Website</label>
                <input />
            </fieldset>
        </div>
    )
};

export default PersonalInfoEntry;
