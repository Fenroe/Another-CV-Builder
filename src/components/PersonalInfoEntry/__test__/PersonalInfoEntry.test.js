import { render, cleanup } from "@testing-library/react";
import React from "react";
import "@testing-library/jest-dom";
import PersonalInfoEntry from "../PersonalInfoEntry";

afterEach(cleanup);

it('renders', () => {
    const {asFragment} = render(<PersonalInfoEntry />);
    expect(asFragment()).toMatchSnapshot();
});
