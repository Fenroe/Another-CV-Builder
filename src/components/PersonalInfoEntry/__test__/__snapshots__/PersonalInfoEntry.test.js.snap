// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <div>
    <fieldset>
      <legend>
        Personal Details
      </legend>
      <label>
        First Name
      </label>
      <input />
      <label>
        Last Name
      </label>
      <input />
      <label>
        Email Adress
      </label>
      <input />
      <label>
        Phone Number
      </label>
      <input />
      <label>
        Website
      </label>
      <input />
    </fieldset>
  </div>
</DocumentFragment>
`;
